catroid/config/lint-baseline.xml

```diff
@@ -116,6 +111,18 @@
             column="51"/>
     </issue>
 
+    <issue
+        id="OutdatedLibrary"
+        message="This version is deprecated. Details: The Fabric Crashlytics SDK is now deprecated and will continue reporting your app&apos;s crashes only until November 15, 2020. To continue getting crash reports in the Firebase console, make sure you upgrade to the Firebase Crashlytics SDK version 17.0.0+.
+&#xA;For more information, visit https://firebase.google.com/docs/crashlytics/upgrade-sdk. Consider switching to recommended version 17.2.2."
+        errorLine1="    implementation (&apos;com.crashlytics.sdk.android:crashlytics:2.6.8@aar&apos;) {"
+        errorLine2="    ^">
+        <location
+            file="build.gradle"
+            line="442"
+            column="5"/>
+    </issue>
+
```

Comment:
```suggestion
```

Fabric & crashlytics were eliminated with #3840 so I believe there's no need for this anymore.